### ISSUE TITLE
agree on a single npm name/repo

### ISSUE THREAD

**Issue by graingert** [nathan-muir/canvgc#3]
Maybe we can all agree on a single repo/npm name https://github.com/gabelerner/canvg/pull/477

**Comment by nathan-muir** [comment]
Yeah, I'll send a PR with my changes, when the opportunity arises.

**Comment by kencaron** [comment]
I'm a little out of context here. Interested in using this command-line CLI tool `canvgc` in a project. Is that tool being merged into the main `canvg` project? Does it already exist? Also, what is [`Painter`](https://github.com/nathan-muir/canvgc/blob/master/README.md#usage-client-side)? Sorry for the 20 questions. :)
